sound/soc/sof/topology.c

```diff
@@ -150,6 +150,11 @@ static int sof_keyword_dapm_event(struct snd_soc_dapm_widget *w,
 
 	/* get runtime PCM params using widget's stream name */
 	spcm = snd_sof_find_spcm_name(sdev, swidget->widget->sname);
+	if (!spcm) {
```

Comment:
when can this happen?
Or to ask differently, how was this error found?

Reply:
It was found by tools from Keqiao :)